binlog_proto/proto/binlog.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto2";
-package binlog;
+package tools.binlog;
```

Comment:
first i use obinlog, change it to binlog when change this pkg to the name binlog_proto, but it will easy conflict with binlog package of tipb, so change it to this here.
it will panic when you use same package name, and then contains the same enum type name

Reply:
how about `slave.binlog`?

Reply:
has change to slave.binlog